packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts

```diff
@@ -177,6 +182,11 @@ export interface IGroupRenderProps {
    * Override which allows the caller to provider a customer footer.
    */
   onRenderFooter?: IRenderFunction<IGroupDividerProps>;
+
+  /**
+   * Flag to indicate whether to ignore the collapsing icon on header.
+   */
+  collapseAllVisibility?: CollapseAllVisibility;
```

Comment:
The current default behavior is visible? If so, then you should add default props to not break existing users